app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -254,12 +225,6 @@ def empty_file?(file)
         (file.respond_to?(:tempfile) && file.tempfile.size == 0) || (file.respond_to?(:size) && file.size == 0)
       end
 
-      # this is provided so that implementing application can override this behavior and map params to different attributes
```

Comment:
@atz Those are admittedly all on Sufia, some older than others.

@jcoyne How much do you care about this change? Based on what I'm seeing in other repos, I'm not too worried about this complicating the CC to Hyrax bridge and thus I'm inclined to stop caring about it in the context of this PR.